e2e_test/test/e2e_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('setup test', () {
```

Comment:
That works. Although I think it'd also be fine to stick the `expectSuccess` inside `setUpAll` and get the same effect